lib/security/WSSecurity.js

```diff
@@ -3,10 +3,11 @@
 var crypto = require('crypto')
   , passwordDigest = require('../utils').passwordDigest;
 
-function WSSecurity(username, password, passwordType) {
+function WSSecurity(username, password, options) {
```

Comment:
@herom take a look at my new updates.  My goal was not to break implementations utilizing the `passwordType` string parameter.  The `options` object is not required but rather optional.  The `passwordType` property default is `PasswordText` and the `hasTimeStamp` property will default to true.